UI/SimpleCalculator/SimpleCalculator.UI/Views/CalculatorMarkupView.cs

```diff
@@ -0,0 +1,253 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using global::System;
+using global::System.Windows.Input;
+using global::Uno.Toolkit.UI;
+using global::Microsoft.UI.Xaml;
+using global::Microsoft.UI.Xaml.Controls;
+using global::Microsoft.UI.Xaml.Controls.Primitives;
+using global::Microsoft.UI.Xaml.Media;
+
+namespace SimpleCalculator.Views
+{
+    public sealed partial class CalculatorMarkupView : Page
+    {
+        public CalculatorMarkupView()
+        {
+            InitMarkupView();
+        }
+
+        private void InitMarkupView()
+        {
+            this
+                .Background(Microsoft.UI.Xaml.Resource.Theme<Brush>("BackgroundBrush"))
```

Comment:
This will need to get updated. Since this needs to be fully qualified on Android this means we need to rename the API.